File: Vehicle-Entry-Exit-Backend-main/src/main/java/com/example/learn1/dto/VehicleDTO.java
package com.example.learn1.dto;

import lombok.Data;
import java.util.List;

@Data
public class VehicleDTO {
    private String vehicleNumber;
    private String vehicleType;
    private String vehicleModelName;
    private List<String> vehicleImages;
    private String vehicleOwner;
    private String driverName;
    private String driverMobileNumber;
    private String empNumber;
}

File: VehicleEntryExit-Android-Fronted--main/app/src/main/java/com/example/vehicleentryexit/models/VehicleDTO.kt
package com.example.vehicleentryexit.models

data class VehicleDTO(
    val vehicleNumber: String,
    val vehicleType: String,
    val vehicleModelName: String,
    val vehicleImages: List<String>,
    val vehicleOwner: String,
    val driverName: String,
    val driverMobileNumber: String,
    val empNumber: String
)
